### DISCUSSION TITLE
Is there a changelog?

### DISCUSSION THREAD

**Discussion by MoeLa** [artilleryio/artillery#1031] [Artillery]
I've found the changelog for [Artillery Pro](https://artillery.io/docs/resources/pro/changelog.html), but not the one for the common version. In particular, like to know, what's new in [v1.7.2](https://www.npmjs.com/package/artillery/v/1.7.2).

**Comment by hassy** [comment]
hi @MoeLa 👋 we're working on an updated Changelog which will include everything in one place. In the meantime, the Releases page on Github provides information on what's new in each release:

https://github.com/artilleryio/artillery/releases
